db/migrate/20190308184016_add_path_column_to_node_version_history_changelog.rb

```diff
@@ -0,0 +1,19 @@
+# 20190308184016_add_path_column_to_node_version_history_changelog.rb
+
+class AddPathColumnToNodeVersionHistoryChangelog < ActiveRecord::Migration
+  include Gitlab::Database::MigrationHelpers
+
+  DOWNTIME = false
+
+  def up
+    if Gitlab::Database.postgresql?
+      execute <<-SQL
+        ALTER TABLE node_version_history_changelog ADD COLUMN path text NOT NULL;
```

Comment:
if this runs on a changelog table with rows in it, this will fail because it has a NOT NULL constraint and no default value. Is there a default we can add? Or you can add the column without the NOT NULL constraint, run an update statement with logic on how to populate the column with real values, and then add a NOT NULL constraint once all rows have this column populated.